### ISSUE TITLE
Please let me add more upgrade cards

### ISSUE THREAD

**Issue by Smiling-Dungeon-Master** [raithos/xwing#1264]
Why can't I add more than 16 points of upgrade cards? I generally like when I have a 50 or 100 point squadron, but I need more points to do this.

**Comment by Patschke** [comment]
I am not sure what this issue is about. You can build arbitrarily large squads (in: as many ships as you want). However the loadout points for any single ship is limited by the pilot you selected. This is due to the rules of the game. What are you asking about? There is nothing like a specific 16 points limit.

**Comment by raithos** [comment]
Yeah I have bad news for you.. if you're coming from 1.0, the way squadbuilding works is different in 2.5. each pilot has a specific loadout of points for upgrades.
